src/features/providers/components/provider-dialog-footer.tsx

```diff
@@ -0,0 +1,17 @@
+import { Button, DialogFooter } from "@stacklok/ui-kit";
+import { useNavigate } from "react-router-dom";
+
+export function ProviderDialogFooter() {
```

Comment:
nit pick, but it would make sense to group all these components (related to the dialog) into a common folder, like `src/features/providers/components/dialog/footer.tsx`. The actual dialog component could be in `index.tsx`

Reply:
you mean in all the app, right?
I created for provider only, a Provider Dialog with all shared dialog components, I guess we can do the same for the rest of the application

Reply:
yeah, I mean I would use this pattern across the app, not just here. The idea is that components that only make sense in a single place (in this case, that footer only makes sense in the provider dialog) could be grouped together in a folder.

Reply:
I think in minder we used this pattern a lot? :thinking: